sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -151,6 +148,17 @@ export class ProgramInfoRequestController {
         payload.pirDenyReasonId,
         payload.otherReasonDesc,
       );
+      const result =
```

Comment:
Did you check if the method _this.applicationService.setDeniedReasonForProgramInfoRequest_ isn't returning the _assessmentWorkflowId_ that is needed?

Reply:
No I did check, let me check

Reply:
Wow, we are getting it, let me change and remove the unwanted method.